validation/provisioning/rke2/hardened_test.go

```diff
@@ -1,11 +1,12 @@
-//go:build (validation || sanity) && !infra.any && !infra.aks && !infra.eks && !infra.rke2k3s && !infra.gke && !infra.rke1 && !cluster.any && !cluster.custom && !cluster.nodedriver && !extended && !stress
+//go:build recurring
```

Comment:
This comment can be taken for all of the subsequent tests. Can you update to: `//go:build validation || recurring`?